dev-python/backports-shutil_which/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+		<email>python@gentoo.org</email>
```

Comment:
Has the Python team agreed to maintain this? They sometimes get a little bit upset when new packages get 'dropped' on them. You're more than welcome to maintain it yourself.